tensorflow/stream_executor/rocm/rocm_blas.h

```diff
@@ -121,7 +131,8 @@ class ROCMBlas : public blas::BlasSupport {
       std::unique_ptr<TemporaryDeviceMemory<
           typename RocBlasTypeConversionHelper<T>::mapped_type>> *temp_memory,
       DeviceMemory<typename RocBlasTypeConversionHelper<T>::mapped_type>
-          *device_memory);
+          *device_memory, bool copy_data,
```

Comment:
I don't think this change belongs here.

Reply:
'copy_data' can be removed, but the rest of the change fixes a bug in the batched GEMM logic.